react/week1/todolist-app/src/components/TodoItem.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
+
+export default class TodoItem extends Component {
+    
+    state =  {
+        todoListItems:[
+            {
+                id: 1,
+                description: 'Get out of bed',
+                deadlineDate: 'Wed Sep 13 2017',
+            },
+            {
+                id: 2,
+                description: 'Brush Teeth',
+                deadlineDate: 'Thu Sep 14 2017',
+            },
+            {
+                id: 3,
+                description: 'Eat breakfast',
+                deadlineDate: 'Fri Sep 15 2017',
+            }
+    ]};        
+
+render() { 
+    return (
+			<ul>             
+        {this.state.todoListItems.map((todo) => (
+                <li key={todo.id}> {todo.description}, {todo.deadlineDate}</li>
+           
+		))}
+		
+			</ul>
+        
+     );
```

Comment:
Here you can break down the component further and make a `Item` component and then pass it the description and the deadline as a prop, fx:
```suggestion
	const TodoList = this.state.todoListItems.map((todo) => (
		<Item
			key={todo.id}
			description={todo.description}
			deadlineDate={todo.deadlineDate}
		/>
	));
	return <ul>{TodoList}</ul>;
```

Reply:
added component 'Item' and implemented the feedback.